src/calculateRentalCost.js

```diff
@@ -29,6 +29,23 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const DAY_COST = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUN = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUN = 20;
+  const priceWithoutDiscount = days * DAY_COST;
+  let price = priceWithoutDiscount;
```

Comment:
I think you only need one constant here, let's call it `basePrice` 
```suggestion
  const basePrice = days * DAY_COST;
```
Then, inside of `if` statements you could return the proper calculation like this:
```
  return basePrice - SHORT_TERM_DISCOUNT;
```